Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -2277,9 +2277,6 @@ theorem omega0_le {o : Ordinal} : ω ≤ o ↔ ∀ n : ℕ, ↑n ≤ o :=
       let ⟨n, e⟩ := lt_omega0.1 h
       rw [e, ← succ_le_iff]; exact H (n + 1)⟩
 
-@[deprecated (since := "2024-09-30")]
-alias omega_le := omega0_le
```

Comment:
We like to deprecate lemmas for a few months. Is there a reason you want to remove this already?

Reply:
This clashed with `omega_le_omega` before that got renamed, I'll add this alias back.